src/App.tsx

```diff
@@ -27,6 +39,7 @@ function App(): React.ReactElement {
               <Link to="/profile">Profile</Link>
             </li>
           </ul>
+          <StyledButton onClick={() => loginWithRedirect()}>Login</StyledButton>
```

Comment:
Probably don't need to create a new function for this 👀
```jsx
<StyledButton onClick={loginWithRedirect}>Login</StyledButton>
```
should be sufficient.